File: CS50X/Python/sentimental-cash/cash.py
from cs50 import get_float

while True:
    cambio = get_float("Cambio: ")
    try:
        if cambio > 0.00:
            break
        else:
            print("Not valid")
    except ValueError:
        print("Not valid")

cambio *= 100

operacion1 = cambio // 25
cambio = cambio % 25
operacion2 = cambio // 10
cambio = cambio % 10
operacion3 = cambio // 5
cambio = cambio % 5
operacion4 = cambio // 1
print(f"Resultado : {operacion1 + operacion2 + operacion3 + operacion4}")

File: CS50X/C/cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{

    int amount, operacion1, operacion2, operacion3, operacion4, resultado;

    do
    {
        amount = get_int("Amount greater than 0: ");
    }
    while (amount <= 0);

    printf("Change owned : %i\n", amount);
    operacion1 = amount / 25;
    amount = amount - 25 * operacion1;
    operacion2 = amount / 10;
    amount = amount - 10 * operacion2;
    operacion3 = amount / 5;
    amount = amount - 5 * operacion3;
    operacion4 = amount / 1;
    amount = amount - 1 * operacion4;
    resultado = operacion1 + operacion2 + operacion3 + operacion4;
    printf("Change:%i\n", resultado);
}
